### PR TITLE
Add FIPS reference to consul enterprise docs

### PR DESCRIPTION
**Pull request by im2nguyen**
### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

File: website/content/docs/enterprise/index.mdx [hunk 1]
```diff
@@ -12,35 +12,51 @@ features address the organizational complexities of collaboration, operations, s
...
 - [Network Areas](/consul/docs/enterprise/federation): Support complex network topologies between federated Consul datacenters with pairwise federation rather than full mesh federation
 - [Network Segments](/consul/docs/enterprise/network-segments/network-segments-overview): Support complex network topologies within a Consul datacenter by enforcing boundaries in Consul client gossip traffic
 
 ### Governance
+
 - [OIDC Auth Method](/consul/docs/security/acl/auth-methods/oidc): Manage user access to Consul through an OIDC identity provider instead of Consul ACL tokens directly
 - [Audit Logging](/consul/docs/enterprise/audit-logging): Understand Consul access and usage patterns by reviewing access to the Consul HTTP API 
 - [Sentinel for KV](/consul/docs/enterprise/sentinel): Policy-as-code framework for defining advanced key-value storage access control policies
 
+## Regulatory compliance
```
Comment (david-yu) [inline website/content/docs/enterprise/index.mdx:50]:
```suggestion
### Regulatory compliance
```

File: website/content/docs/enterprise/index.mdx [hunk 2]
```diff
@@ -12,35 +12,51 @@ features address the organizational complexities of collaboration, operations, s
...
+
+- [FIPS 140-2 Compliance](/consul/docs/enterprise/fips): Leverage FIPS builds of Consul Enterprise to ensure your Consul deployments are secured with BoringCrypto and CNGCrypto, and compliant with FIPS 140-2.
+
+  <Note>
+
+  FIPS 140-2 builds of Consul Enterprise support all runtimes (VMs, Kubernetes, ECS) except for Lambda. In addition, HCP does not currently support FIPS builds of Consul Enterprise.
```
Comment (david-yu) [inline website/content/docs/enterprise/index.mdx:56]: I'll leave it up for @Jeff-Apple to confirm but I believe ECS does not support FIPS yet.

**Review by david-yu — APPROVED** [review]
I think we can remove ECS for now, and add it back in once Jeff confirms if it does.

File: website/content/docs/enterprise/index.mdx (hunk 2 above)
Comment (Jeff-Apple) [reply website/content/docs/enterprise/index.mdx:56]: I'm waiting to hear back from the engineer as to why we haven't published the FIPS version of consul-ecs yet. It is supposed to be supported.

Comment (Jeff-Apple) [reply website/content/docs/enterprise/index.mdx:56]: ECS isn't supported for now. It will be the next time we release a new version of consul-ecs.

**Review by Jeff-Apple — APPROVED** [review]

**Pull request merged**